typescript/graphql-apollo-server/schema.graphql

```diff
@@ -3,10 +3,8 @@
 
 
 type Mutation {
-  createDraft(authorEmail: String, content: String, title: String!): Post!
-  deleteOnePost(where: PostWhereUniqueInput!): Post
+  createDraft(authorEmail: String!, content: String, title: String!): Post!
   publish(id: Int): Post
-  signupUser(data: UserCreateInput!): User!
```

Comment:
What are all these deletions in this file?

Reply:
I didn't make the change, it is most likely a "generated" change

Reply:
This should be understood then before committing - it changes the example and might be wanted - or not.

Reply:
`"generate:nexus": "ts-node --transpile-only src/schema"` this is in the `package.json` file, I think this is generated. I call @nikolasburk to validate.